.happy/terraform/modules/ecs-stack/main.tf

```diff
@@ -24,20 +24,21 @@ locals {
   backend_cmd                  = ["gunicorn", "--worker-class", "gevent", "--workers", "1", "--bind", "0.0.0.0:5000", "backend.corpora.api_server.app:app", "--max-requests", "10000", "--timeout", "30", "--keep-alive", "5", "--log-level", "info"]
   data_load_path               = "s3://${local.secret["s3_buckets"]["env"]["name"]}/database/dev_data.sql"
 
-  vpc_id                       = local.secret["vpc_id"]
-  subnets                      = local.secret["private_subnets"]
-  security_groups              = local.secret["security_groups"]
-  zone                         = local.secret["zone_id"]
-  cluster                      = local.secret["cluster_arn"]
-  frontend_image_repo          = local.secret["ecrs"]["frontend"]["url"]
-  backend_image_repo           = local.secret["ecrs"]["backend"]["url"]
-  upload_image_repo            = local.secret["ecrs"]["processing"]["url"]
-  lambda_upload_success_repo   = local.secret["ecrs"]["upload_success"]["url"]
-  lambda_upload_repo           = local.secret["ecrs"]["upload_failures"]["url"]
-  batch_role_arn               = local.secret["batch_queues"]["upload"]["role_arn"]
-  job_queue_arn                = local.secret["batch_queues"]["upload"]["queue_arn"]
-  external_dns                 = local.secret["external_zone_name"]
-  internal_dns                 = local.secret["internal_zone_name"]
+  vpc_id                         = local.secret["vpc_id"]
+  subnets                        = local.secret["private_subnets"]
+  security_groups                = local.secret["security_groups"]
+  zone                           = local.secret["zone_id"]
+  cluster                        = local.secret["cluster_arn"]
+  frontend_image_repo            = local.secret["ecrs"]["frontend"]["url"]
+  backend_image_repo             = local.secret["ecrs"]["backend"]["url"]
+  upload_image_repo              = local.secret["ecrs"]["processing"]["url"]
+  lambda_upload_success_repo     = local.secret["ecrs"]["upload_success"]["url"]
+  lambda_upload_repo             = local.secret["ecrs"]["upload_failures"]["url"]
+  lambda_dataset_submission_repo = local.secret["ecrs"]["dataset_submission"]["url"]
```

Comment:
New ECR repo for dataset submission lambda. The rest is reformatting due to goofy TF convention.